.gitignore

```diff
@@ -119,6 +119,7 @@ ENV/
 
 # Emacs
 \#*#
+.emacs.desktop*
```

Comment:
Is that meant to be in this PR?

Reply:
I can file another for it.

Reply:
Up to the maintainers but that would be my suggestion if it's unrelated. :)

Reply:
Removed.